packages/build/src/core/main.js

```diff
@@ -71,19 +71,23 @@ const build = async function (flags = {}) {
       framework,
       dry,
       siteInfo,
-      telemetry,
       mode,
       logs,
       timers,
       durationNs,
       testOpts,
       statsdOpts,
```

Comment:
Several of the arguments passed to `handleBuildSuccess()` can probably be removed now.

Reply:
Actually I think all the other ones are required for the `statsd` timers.

Reply:
I might be missing something, but aren't the parameters removed in https://github.com/netlify/build/pull/2349/files#diff-55e26a417d07c8f3e5e6079bf2187f41952d21ac781fc937a57d6711f8f54734R433 ?

Reply:
You're right, sorry, I totally misunderstood the review 🤦 addressed in https://github.com/netlify/build/pull/2349/commits/98e817ce6a718eb95deee5652c81f4709d22fb8c